DQMOffline/Trigger/src/HLTTauDQMPath.cc

```diff
@@ -197,13 +197,16 @@ namespace {
 
   int getParameterSafe(const HLTConfigProvider& HLTCP,
                        const std::string& filterName,
-                       const std::string& parameterName) {
+                       const std::string& parameterName,
+                       bool verbose) {
     const edm::ParameterSet& pset = HLTCP.modulePSet(filterName);
     if (pset.existsAs<int>(parameterName))
       return pset.getParameter<int>(parameterName);
     else {
-      edm::LogWarning("HLTTauDQMOfflineSource") << "No parameter '" << parameterName << "' in configuration of filter "
-                                                << filterName << " pset " << pset.dump() << std::endl;
+      if (verbose)
```

Comment:
@slehti @jfernan2 sorry, isn't this exercise a complex way to downgrade a LogWarning to a LogInfo?

Reply:
@slehti @jfernan2 ping

Reply:
I did not want to change any LogWarnings into LogInfo, only to have a way to silence some warnings. And to have them available when needed. We need to fix a long standing problem in many RelVal workflows coming from HLTTau DQM modules: warnings are flooding the logs (since a long time).

Reply:
Timeline: as soon as possible.

Reply:
Then I believe @fabiocos is right: we should not sweep under the carpet